### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by Corei13**

### CODE REVIEW

File: update.js [hunk 1]
```diff
@@ -1,18 +1,19 @@
-
 const NavBar: FC = () => {
     const [pageSwtichButtons, setPageSwitchButtons] = useState(
       InitialPageSwitchButtons
     );
-    const [currentButton, setCurrentButton] = useState(
-      InitialPageSwitchButtons[0]
-    );
-  
+dfsdfsd
```
Comment (Corei13) [inline update.js:5]: This is test comment

Comment (Corei13) [reply update.js:5]: This is reply

File: update.js [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
     const [pageSwtichButtons, setPageSwitchButtons] = useState(
       InitialPageSwitchButtons
     );
-    const [currentButton, setCurrentButton] = useState(
-      InitialPageSwitchButtons[0]
-    );
+dfsdfsd
   
     return (
         <div className="d-flex flex-row align-items-center navbar-desktop-right-content">
```
Comment (pritom-dip) [inline update.js:11]: "test"

Comment (pritom-dip) [inline update.js:11]: PullRequest
This is the main comment
 "test" 
 Sample Reply

File: update.js [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
           {pageSwtichButtons.map((button) => {
-                      alt=""
                       className="navbar-page-switch-button-icon"
+                      kjkjnkjn
                     />
+                    kjnkjn
                     {button.name}
```
Comment (pritom-dip) [inline update.js:17]: PullRequest
This is the main comment
 "test" 
 Sample Reply

File: update.js (hunk 2 above)
Comment (pritom-dip) [inline update.js:11]: <div><div>It seems that your code expects the byte array to represent a 32bit integer in big-endian byte order. We assume that for some reason if you run your code under linux, the byte order is little-endian. We would need to know where the byte array comes from to guess a reason for this.<br/><br/></div><div><b>Name</b> <sub>Just Now</sub><br/>Good Catch! I will go ahead and change this now. For now, we are not worried about endian, We will change it after. Thanks for the suggestion. Do you have any recommendations for os specific task that we may need to change further? I've been thinking about shifting some of our codebases to cloud services. </div></div>

Comment (pritom-dip) [inline update.js:11]: <div><div>It seems that your code expects the byte array to represent a 32bit integer in big-endian byte order. We assume that for some reason if you run your code under linux, the byte order is little-endian. We would need to know where the byte array comes from to guess a reason for this.<br/><br/></div><div><b>Name</b> <sub>Just Now</sub><br/>Good Catch! I will go ahead and change this now. For now, we are not worried about endian, We will change it after. Thanks for the suggestion. Do you have any recommendations for os specific task that we may need to change further? I've been thinking about shifting some of our codebases to cloud services Also Is there any alternative to google Map API for showing maps and nearest places?</div></div>

**Pull request closed without merging**